segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/bookkeeper/BookKeeperServiceRunner.java

```diff
@@ -214,6 +215,12 @@ private BookieServer runBookie(int bkPort) throws Exception {
         conf.setAllowLoopback(true);
         conf.setJournalAdaptiveGroupWrites(false);
         conf.setZkLedgersRootPath(ledgersPath);
+        conf.setTLSProvider("OpenSSL");
```

Comment:
Do we need to do this for the internal tests too?

Reply:
I think you mean `integration` tests. We do not need it right now as the path is already covered in unit tests.

Reply:
If we do not need it right now, then can we get rid of it? I do not like hardcoding these paths in the code, since they may change.

Reply:
We need it for the tests with secure BookKeeper to work. In case of non-secure BK this is just a fallthrough. Also this is just a string, not a path.